nvme-print.c

```diff
@@ -490,9 +491,24 @@ void nvme_show_single_property(int offset, uint64_t value64, nvme_print_flags_t
 	nvme_print(single_property, flags, offset, value64);
 }
 
-void nvme_show_relatives(const char *name)
+void nvme_show_relatives(const char *name, nvme_print_flags_t flags)
 {
-	/* XXX: TBD */
+	int err = 0;
+
+	_cleanup_nvme_root_ nvme_root_t r = nvme_create_root(stderr, log_level);
+
+	if (!r) {
+		nvme_show_error("Failed to create topology root: %s", nvme_strerror(errno));
+		return;
+	}
+
+	err = nvme_scan_topology(r, NULL, NULL);
```

Comment:
Instead creating a new root and scan sysfs again, just pass in the `nvme_root_t` object into `nvme_show_relatives`.